### ISSUE TITLE
pop up when creating a new record after upgrade to jq v6

### ISSUE THREAD

**Issue by cpg** [activescaffold/active_scaffold#529]
FYI. I am using AS master after I updated my bundle and updating to jqiery-6.

Things are working as usual for a couple of days, however, today I noticed that I get a pop up when I'm creating a new records. There is a "Close" button, however, I do notice that the record appears to be created properly.
```
RJS error:
TypeError: undefined is not an object (evaluating 'effectMethod.mode')
```
Is this an AS bug you guys should be aware of, or something I need to address after my upgrade?

**Comment by scambra** [comment]
are you able to get backtrace of JS? So I can see what line of AS js is raising exception, probably some JS needs to be updated for newer jQuery

**Comment by cpg** [comment]
Here is what comes out in the console while I get that pop-up. Does it help?
(again, using master as of yesterday) 

```
[Error] TypeError: undefined is not an object (evaluating 'effectMethod.mode')
	effect (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:13644)
	highlight (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:24353)
	insert (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:24904)
	(anonymous function) (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:23648)
	dispatch (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:5227)
	handle (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:4879)
	trigger (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:5131)
	(anonymous function) (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:5861)
	each (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:371)
	each (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:138)
	trigger (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:5860)
	success (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:11166)
	fire (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:3233)
	fireWith (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:3363)
	done (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:9841)
	callback (cpanel-6b257311df958d112f248e8df96f2c122fbc7e2c317602c632dc552483df4a82.js:10312)
```

**Comment by cpg** [comment]
Another thing which I notice is that the search field does not focus upon coming up, so I looked up in the error console and I saw this same error. There is no pop-up in this case, however.

**Comment by scambra** [comment]
It may help, but it would be better to get trace from development mode, so I see source files and lines numbers

**Comment by cpg** [comment]
I am out for the time being and not able to get on this for a few days. However, I am not doing anything too sophisticated, so any AS project you may have around should be able to reproduce it, I would imagine.

Today I got an unusual issue with search, though it's on the rails side, so I am not sure if it's related, however, this has been working all along.

**Comment by scambra** [comment]
jquery-ui 1.12 (included in jquery-ui-rails 6.0 gem) has moved effect to folder, so specific effect needs to be loaded beside effects.js file. I have fixed it, but ActiveScaffold only uses highlight effect, if you have overrided highlight method to use different effect, or has custom code using different effects, you have to add them to your manifest.
